src/sentry/static/sentry/app/views/organizationIncidents/details/header.jsx

```diff
@@ -36,17 +35,7 @@ export default class DetailsHeader extends React.Component {
         access={['org:write']}
         renderNoAccessMessage={() => <Status incident={incident} />}
       >
-        <DropdownControl
-          button={
-            // eslint-disable-next-line no-shadow
-            ({getActorProps, isOpen}) => (
-              <DropdownButton {...getActorProps()} isOpen={isOpen}>
```

Comment:
By using `label` prop, we can use default styling of `DropdownButton` that `DropdownControl` has.